plasmapy/addons/__init__.py

```diff
@@ -0,0 +1,28 @@
+__all__ = []
+
+from pkg_resources import (iter_entry_points, resource_filename)
+
+
+def _parse_readme():
+    fname = resource_filename('plasmapy', 'addons/README.md')
```

Comment:
I take it this handles wheel installs, etc? Nice! I'll be coming back here often :laughing:

Reply:
Yes it does.  I actually forgot about this. 😆

From [`pkg_resources` documentation](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#resource-extraction)...

> resource_filename(package_or_requirement, resource_name)
>
>     Sometimes, it is not sufficient to access a resource in string or stream form, and a true 
>     filesystem filename is needed. In such cases, you can use this method (or module-level 
>     function) to obtain a filename for a resource. If the resource is in an archive distribution 
>     (such as a zipped egg), it will be extracted to a cache directory, and the filename within 
>     the cache will be returned. If the named resource is a directory, then all resources within 
>     that directory (including subdirectories) are also extracted. If the named resource is a C 
>     extension or “eager resource” (see the setuptools documentation for details), then all C 
>     extensions and eager resources are extracted at the same time.